crates/starknet/src/starknet/starknet_config.rs

```diff
@@ -57,6 +58,7 @@ impl Default for StarknetConfig {
             dump_on: None,
             dump_path: None,
             re_execute_on_init: true,
+            state_archive: false,
```

Comment:
discussing with some projects that use devnet-rs, for now it seems the default can be `false`